spec/oas_v1.yaml

```diff
@@ -0,0 +1,641 @@
+          type: boolean
+        twitter:
+          description: Optional twitter handle for a particular delegate.
+          type: string
+          examples:
+            - summary: Twitter handle
+              value: "@flip_liquide"
+        statement:
+          description: A statement from the delegate describing their voting positions.
+          type: string
+          examples:
+            - summary: Delegate statement
+              value: "hello! I am a great delegate for reasons xyz"
+        topIssues:
+          description: The most important issues for a given delegate.
+          type: array
+          items: 
+              $ref: '#/components/schemas/Issue'
+    Issue:
+      summary: Description of a governance issue
+      description: An object describing a particular view on a particular governance issue.
+      type: object
+      properties:
+        type: 
+          type: string
+          examples: 
+            - "Funding"
+        value:
+          type: string
+          examples: 
+            - "Grant funding should be directed to projects with a great track record"
+    SnapshotProposalData:
+      summary: Data associated with a snapshot proposal
+      description: >
+        Metadata and data associated with a snapshot proposal, including start/end/created times, scores, and votes. 
+      type: object
+      properties:
+        votingStrategy: 
+          $ref: "#/components/schemas/VotingStrategy"
+        title:
+          summary: Title of the proposal
+          type: string
+        startTimestamp:
+          summary: Start time of the proposal
+          type: string
+          format: date-time
+        endTimestamp:
+          summary: End time of the proposal
+          type: string
+          format: date-time
+        createdTimestamp:
+          summary: Creation time of the proposal
+          type: string
+          format: date-time
+        link:
+          summary: Link to the proposal
+          type: string
+        scores: 
+          summary: Scores for the proposal
+          type: array
+          items:
+            type: string 
+        votes:
+          summary: Votes for the proposal
+          type: string
+        state: 
+          summary: State of the proposal
+          type: string
+          enum: 
+            - ACTIVE
+            - CLOSED
+            - PENDING
+    StandardProposalData: 
+      summary: Data associated with standard proposal 
+      description: >
+        Metadata and data associated with a standard proposal, including onchain execution data.
+      type: object
+      properties:
+        votingStrategy: 
+          $ref: "#/components/schemas/VotingStrategy"
+        options:
+          type: array 
+          items:
+            type: object 
+            properties:
+              executionData:
+                $ref: "#/components/schemas/ExecutionData"
+              signatures:
```

Comment:
`signatures` in the context of proposal are functionSignatures which are basically `functionData` that you already have under `ExecutionData`

Reply:
great catch